### DISCUSSION TITLE
실행 컨텍스트 스택이 가득 차면 어떤 일이 발생하나요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#83] [실행컨텍스트]
- 실행 컨텍스트 스택이 가득 차면 어떤 일이 발생하나요?

**Comment by pipisebastian** [comment]
> 실행 컨텍스트 스택이 가득 차면 어떤 일이 발생하나요?

- 스택이 계속 쌓여, 오버플로우가 발생합니다. 예를 들어 재귀함수를 계속 호출하는 경우가 있습니다.

**Comment by publdaze** [comment]
> - 실행 컨텍스트 스택이 가득 차면 어떤 일이 발생하나요?
- 스택이 가득차면 스택 오버플로우가 발생함, 재귀함수를 무한히 호출할 경우에 주로 발생
